doc/api/next_api_changes/behavior/19128-AG.rst

```diff
@@ -0,0 +1,15 @@
+Harmonized key event data across backends
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Seems like this maybe should be consolidated with `doc/api/next_api_changes/behavior/17791-AL.rst`.